app/views/schools/users/show.html.erb

```diff
@@ -53,9 +53,12 @@
           <p>The user will be sent an email to create an account and will have access to the school's dashboard.</p>
         </div>
 
+        <div class="govuk-button-group">
           <%= f.govuk_submit 'Accept and submit', data: { prevent_double_click_target: "submitButton" } %>
+          <%= link_to "Cancel",  schools_users_path, 'aria-label': "Cancel user invite", secondary: true %>
```

Comment:
I can see that this commit is called 'Change cancel link to a button'. This update changes it to a link and not a button - it should be `govuk_link_to` if we want a button. However, this raises an interesting question of how we should be approaching 'secondary' actions. The GOV design system approaches 'cancel' actions as a secondary link:
![image](https://github.com/DFE-Digital/schools-experience/assets/35870975/914276c5-8b74-4f90-aca4-354b83c1655a)

I think we should try and be consistent with this. We currently have one page on this flow as a 'cancel' button and one as a 'cancel' link.

I suggest we change our other page (app/views/schools/users/_form.html.erb, line 23) to the same approach as this one, an anchor link.

cc @gemmadallmandfe.

Reply:
That sounds sensible to me - we should follow the design system

Apologies @ekumachidi if we've gone in circles a bit on this!